core/src/main/java/nu/yona/server/analysis/entities/ActivityCommentMessage.java

```diff
@@ -76,6 +80,10 @@ public void clearThreadHeadSelfReference()
 	public void setBuddyMessage(ActivityCommentMessage message)
 	{
 		buddyMessage = message;
+		if (message == null)
```

Comment:
Is this actually occurring now? I think `prepareForDelete` already sets to `null`?